aiokatcp/aiokatcp_server.py

```diff
@@ -0,0 +1,44 @@
+"""This module contains a class called MyServer for creating an aiokatcp server."""
+
+import aiokatcp
+import asyncio
+import datetime
+
+class MyServer(aiokatcp.DeviceServer):
+    """This class inherits from the aiokatcp DeviceServer class.
+
+    This class is used to create an aiokatcp server object.
+    """
```

Comment:
Be a bit more descriptive in your documentation what your derived class actually adds, or why it's created. Saying that it's inheriting from DeviceServer adds little value to your documentation because I can see in the line just above that this is what is happening. I'd like to know what you do in this derived class that the base doesn't do.

In this case, it's part of a training exercise so there's no purpose, but it may be something along the lines of "Implement a control and monitoring interface for a correlator."

My suggestion of how to implement this in your specific case would look something like this perhaps:
```suggestion
    """Example katcp server derivation for training purposes.

     Some example requests are added to the base class to demonstrate the format for the
     benefit of the trainee.
    """
```